src/components/projectsettings.vue

```diff
@@ -453,6 +460,17 @@ export default defineComponent({
       },
     });
 
+    const membersCanPerformRunAction: Ref<boolean> = computed({
```

Comment:
is this really needed?

Reply:
I followed the same logic as 'isPrivate' checkbox.

Reply:
You are right. I fixed it.

Thank you.